### PR TITLE
Fix some whitespace

### PR DESCRIPTION
**Pull request by yosifkit**

### CODE REVIEW

File: 6.0/Dockerfile [hunk 1]
```diff
@@ -21,14 +21,14 @@ RUN docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr \
...
-    echo "opcache.fast_shutdown=1"; \
-    echo "opcache.enable_cli=1"; \
-    ) > /usr/local/etc/php/conf.d/opcache-recommended.ini
+RUN { \
+		echo "opcache.memory_consumption=128"; \
+		echo "opcache.interned_strings_buffer=8"; \
+		echo "opcache.max_accelerated_files=4000"; \
+		echo "opcache.revalidate_freq=60"; \
+		echo "opcache.fast_shutdown=1"; \
+		echo "opcache.enable_cli=1"; \
```
Comment (tianon) [inline 6.0/Dockerfile]: Wanna switch to single quotes too?

Comment (yosifkit) [inline 6.0/Dockerfile]: Done!

**Comment by lazyfrosch** [comment]
Oh tabs I see, will remember that :)

**Comment by tianon** [comment]
LGTM

@lazyfrosch yeah, we try to avoid making contributors round trip for just pedantic stuff like this unless they're already needing to tweak something else about the PR :+1:

**Pull request merged**